### ISSUE TITLE
btn--ghost broken on safari/webkit

### ISSUE THREAD

**Issue by pds2208** [ONSdigital/design-system#979]
The 'Sign Out' button with the embedded SVG shows up as all white on webkit browsers.

This is due to the tag '-webkit-appearance: button''  in main.css:

[type=reset], [type=submit], button, html [type=button]

<img width="529" alt="Screenshot 2020-09-10 at 10 27 45" src="https://user-images.githubusercontent.com/22938608/92714815-96248680-f354-11ea-9521-f289b3157948.png">

Looking at the design page here: https://ons-design-system.netlify.app/components/header/examples/external-lang/ and all looks good. However this is because of the following error shown in the console:

<img width="1790" alt="Screenshot 2020-09-10 at 11 03 02" src="https://user-images.githubusercontent.com/22938608/92715320-3c708c00-f355-11ea-830f-ea13fb8fddb3.png">

So basically it works on the preview page because the preview page is broken.

If I add the following to the button components it shows up correctly:

-webkit-appearance: none on my component it works as expected.

**Comment by rmccar** [comment]
Hi @pds2208 I've just been trying to recreate this error on Safari. Ive removed the livereload error and I still cant seem to get the error you are talking about. Is there anything else you're doing to create it?

**Comment by pds2208** [comment]
Nope.

**Comment by rmccar** [comment]
Unable to recreate this issue so closing this issue
